cookbook/persistance/key-value.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: page
+title: "Storing key-value data on disk"
+permalink: /cookbook/persistence/key-value/
+---
+
+If you have a relatively small collection of key-values that you'd like to save, you can use the [shared_preferences](https://pub.dartlang.org/packages/shared_preferences) plugin.
+
+Normally we would have to write native platform integration for storing data for both platforms, but fortunately [shared_preferences](https://pub.dartlang.org/packages/shared_preferences) plugin can be used for that. Shared preferences plugin wraps `NSUserDefaults` on iOS and `SharedPreferences` on Android, providing a persistent store for simple data.
```

Comment:
"... for both platforms. Fortunately, the shared_prefs..." 

Two sentences: "but fortunately" is a bit awkward;  put "the" before shared_preferences